src/main/java/org/broadinstitute/hellbender/tools/dataflow/transforms/bqsr/BaseRecalibratorFn.java

```diff
@@ -191,10 +200,16 @@ private void apply(GATKRead originalRead, final ReferenceDataSource refDS, Itera
         }
 
         // We've checked in onTraversalStart() that we have a reference, so ref.get() is safe
-        final int[] isSNP = calculateIsSNP(read, refDS);
-        final int[] isInsertion = calculateIsIndel(read, EventType.BASE_INSERTION);
-        final int[] isDeletion = calculateIsIndel(read, EventType.BASE_DELETION);
-        final int nErrors = nEvents(isSNP, isInsertion, isDeletion);
+        //final int[] isSNP = calculateIsSNP(read, refDS);
+        //final int[] isInsertion = calculateIsIndel(read, EventType.BASE_INSERTION);
+        //final int[] isDeletion = calculateIsIndel(read, EventType.BASE_DELETION);
+        //final int nErrors = nEvents(isSNP, isInsertion, isDeletion);
```

Comment:
The minor refactoring you did in this class conflicts with the BQSR engine refactoring we did in https://github.com/broadinstitute/hellbender/tree/dr_lb_spark_bqsr -- if there are any changes here that must be preserved for **performance** reasons, please let us know.